go.mod

```diff
@@ -9,8 +9,12 @@ require (
 	github.com/golang/protobuf v1.3.1
 	github.com/google/go-cmp v0.3.0
 	github.com/iovisor/gobpf v0.0.0-20191024162143-7c8f8e040b4b
+	github.com/json-iterator/go v1.1.6 // indirect
```

Comment:
Hmm... BTW are you still compiling with golang < 1.13, or is this just one of those cases where go mod for some reason still inserts indirect dependencies (also) to go.mod ?

Reply:
I'm using Go 1.13 (on Clear Linux). I'll see if I can clean this up.

Reply:
Don't worry, probably no need for it. If you're already on 1.13 then my guess is it's just go mod being buggy. We were wondering about a very similar case with @marquiz  the other day. Eventually we couldn't figure out how to convince go mod not to add a few indirect dependencies to go.mod.